packages/calcite-components/src/components/switch/switch.scss

```diff
@@ -3,23 +3,38 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-switch-corner-radius: Specifies the corner radius of the component.
  * @prop --calcite-switch-handle-background-color: Specifies the background color of the component handle.
  * @prop --calcite-switch-handle-border-color: Specifies the border color of the component handle.
  * @prop --calcite-switch-track-background-color: Specifies the background color of the component track.
  * @prop --calcite-switch-track-border-color: Specifies the border color of the component track.
- * @prop --calcite-switch-corner-radius: Specifies the corner radius of the component.
- * @prop --calcite-switch-shadow: Specifies the shadow of the component.
+ * @prop --calcite-switch-handle-shadow: Specifies the shadow of the component handle.
+ * @prop --calcite-switch-handle-border-color-hover: Specifies the border color of the component handle when hovered.
+ * @prop --calcite-switch-shadow-hover: Specifies the shadow of the component handle when hovered.
```

Comment:
Should these be set by the user, for instance `calcite-switch:hover { --calcite-switch-handle-border-color: purple }`? I think the hover/active/stateful props are only for inaccessible components within the shadowRoot. Unless I missed a convention change (which.. very possible).

Reply:
I meant to reply sooner, but this is related to a recent guideline update:

> component tokens should be added for the appropriate states as well. I.E. if background-color changes on hover then you should also add a `background-color-hover` token.

Reply:
Gotcha. In that case, the css prop makes sense - but maybe the description is confusing? Really it's changing the handle-border-color when the entire switch is hovered, not only when the handle is hovered - maybe "... of the component's handle when the component is hovered" ? Kind of verbose, but more accurate.

Reply:
Oh, good catch. I'll double check prop descriptions. There might be some copy pasta afoot. ©️🍝🦶

Reply:
Updated prop descriptions. We'll need to audit the doc once we're all components are updated for clarity and consistency too. cc @geospatialem @DitwanP